Libraries/Opc.Ua.Server/Subscription/MonitoredItem.cs

```diff
@@ -303,6 +317,21 @@ public bool IsReadyToTrigger
             }
         }
 
+        /// <summary>
+        /// Gets or Sets a value indicating whether the item is in ResendData state
+        /// </summary>
+        public int IsResendData
```

Comment:
sounds like a bool variable

Reply:
Yes, (interlock could not operate on bools, but will change to bool since the interlock is useless as you pointed out)